libraries/c_sdk/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -44,6 +44,12 @@
 #include "platform/iot_clock.h"
 #include "platform/iot_threads.h"
 
+/*MQTT v4_beta_2 library includes.*/
+#include "mqtt_config.h"
```

Comment:
we may not need the mqtt_config here. This config is only needed by the new MQTT library.